media/js/newsletter/newsletter-init.es6.js

```diff
@@ -4,7 +4,7 @@
  * file, You can obtain one at https://mozilla.org/MPL/2.0/.
  */
 
-import MzpNewsletter from '@mozilla-protocol/core/protocol/js/newsletter';
+import MzpNewsletter from './newsletter-aws.es6.js';
```

Comment:
So this file will be used for both basket and AWS? I was confused by the naming. I'm not sure we need the AWS namespace here. It might be enough to drop the Protocol one.